crates/rspack_plugin_progress/src/lib.rs

```diff
@@ -54,54 +149,40 @@ impl Plugin for ProgressPlugin {
     _compilation: &mut Compilation,
     _param: &mut MakeParam,
   ) -> PluginMakeHookOutput {
-    self.progress_bar.reset();
-    self.progress_bar.set_prefix(
-      self
-        .options
-        .prefix
-        .clone()
-        .unwrap_or_else(|| "Rspack".to_string()),
-    );
+    if self.options.profile.unwrap_or(false) == false {
+      self.progress_bar.reset();
+      self.progress_bar.set_prefix(
+        self
+          .options
+          .prefix
+          .clone()
+          .unwrap_or_else(|| "Rspack".to_string()),
+      );
+    }
+    self.handler(0.01, String::from("make"), vec![]);
     self.modules_count.store(0, SeqCst);
     self.modules_done.store(0, SeqCst);
-    self.progress_bar.set_message("make");
-    self.progress_bar.set_position(1);
     Ok(())
   }
 
   async fn build_module(&self, module: &mut dyn Module) -> Result<()> {
     if let Some(module) = module.as_normal_module() {
       self
-        .progress_bar
-        .set_message(format!("building {}", module.raw_request()));
-    } else {
-      self.progress_bar.set_message("building");
+        .last_active_module
+        .write()
+        .expect("TODO:")
+        .replace(module.raw_request().to_string());
     }
     self.modules_count.fetch_add(1, SeqCst);
+    self.update_throttled();
     Ok(())
   }
-
-  async fn succeed_module(&self, _module: &dyn Module) -> Result<()> {
```

Comment:
Why delete these? I know that webpack didn't hook succeedModule in ProgressPlugin, maybe we can change this to finishMake

Reply:
build_module and succeed_module has little difference. What do you mean about `finishMake`?

Reply:
@ahabhgk Can you resolve my confusion if you are free?

Reply:
Any reason why delete these?

Reply:
Since build_module used to be used for `set_message` and `succeed_module` used to be used for `set_position`, now that it's been merged into `update`, `succeed_module` seems unnecessary.